src/System.Private.ServiceModel/tests/Common/src/BridgeClient.cs

```diff
@@ -21,20 +22,26 @@ private static string BridgeBaseAddress
             get { return TestProperties.GetProperty(TestProperties.BridgeUrl_PropertyName); }
         }
 
-        static BridgeClient()
+        private static void EnsureBridgeIsRunning()
         {
```

Comment:
Better behavior for error handling.  As a static ctor, we got TypeInitialization exceptions for the BridgeClient that sort of masked something as simple as the bridge not running.  With an explicit failure from EnsureBridgeIsRunning, the exception stack trace is more informative.